frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx

```diff
@@ -329,4 +335,22 @@ describe('MetricsStepper', () => {
 
     expect(exportToJsonFile).toHaveBeenCalledWith(expectedFileName, expectedJson)
   }, 50000)
+
+  it('should clean the config information that is hidden when click next button', async () => {
+    const { getByText } = setup()
+
+    await fillConfigPageData()
+    await userEvent.click(getByText(NEXT))
+
+    expect(updateBoard).not.toHaveBeenCalledWith({
+      type: BOARD_TYPES.JIRA,
+      boardId: '',
+      email: '',
+      projectKey: '',
+      site: '',
+      token: '',
+    })
+    expect(updateSourceControl).toHaveBeenCalledWith({ type: SOURCE_CONTROL_TYPES.GITHUB, token: '' })
+    expect(updatePipelineTool).toHaveBeenCalledWith({ type: PIPELINE_TOOL_TYPES.BUILD_KITE, token: '' })
+  }, 50000)
```

Comment:
can we remove the timeout(`50000`)

Reply:
> can we remove the timeout(`50000`)

Nope, it would make the test fail. It seems like need time to load the element to execute fillConfigPageData function